pkg/crc/network/proxy.go

```diff
@@ -81,8 +81,8 @@ func (p *ProxyConfig) HttpsProxyForDisplay() string {
 }
 
 // AddNoProxy appends the specified host to the list of no proxied hosts.
-func (p *ProxyConfig) AddNoProxy(hosts string) {
-	hosts = strings.Replace(hosts, " ", "", -1)
+func (p *ProxyConfig) AddNoProxy(host ...string) {
+	hosts := strings.Join(host, ",")
 	p.NoProxy = fmt.Sprintf("%s,%s", hosts, p.NoProxy)
 }
```

Comment:
That's a bit odd as this method was changed in the previous commit, and is somehow changed back here.
Might also work as
```
host = append(host, p.NoProxy)
p.NoProxy = strings.Join(host, ",")
```

Reply:
@cfergeau this will only work if we take `host` as slice instead of string.